resources/views/backend/hotels/show.blade.php

```diff
@@ -69,7 +72,7 @@
           @endforeach
         </ul>
         <div class="clearfix"></div>
-        <h3>{{ __('Rooms') . ' (' . $hotel->rooms->count() . ')' }}</h3>
+        <h3><a href="{{ route('room.index', $hotel->id) }}">{{ __('Rooms') . ' (' . $hotel->rooms->count() . ')' }}</a></h3>
```

Comment:
I think we should show text like this: `Total rooms: 8.`
And please be careful between `$hotel->rooms->count()` and `$hotel->rooms()->count()`.
1. `$post->comments->count();` loads the collection and returns count of its elements – redundant if all you need is the count.
2. `$post->comments()->count();` better, no collection loading, but runs the query everytime we use it.